### ISSUE TITLE
Update Debian packaging to latest best practices and debian policy

### ISSUE THREAD

**Issue by mika** [grml/grml#114]
Especially in packages like grml-autoconfig + grml-etc-core we have >10 year old stuff which should be replaced with best practices and following current version of [Debian policy](https://www.debian.org/doc/debian-policy/). While at it try to get rid of backwards compatibility that's no longer relevant and ensure the packages are lintian clean.

**Comment by zeha** [comment]
@mika did you have something specific in mind for grml-etc-core and grml-autoconfig?

You did quite some cleanup this year. If there is nothing actionable left for now, I'd suggest closing this issue.

Specific tasks for grml-etc-core and grml-autoconfig could live in new issues, if any.

**Comment by zeha** [comment]
@mika assigned you for following up on my question above. Don't take it as a "you have to do all the work" assignment please :)

**Comment by mika** [comment]
@zeha yeah, grml-etc-core should get minimized `debian/rules`, and maybe we can look at all our packages together and see which ones might also be targeted towards main Debian (I recall users + customers who are interested in e.g. `grml-hwinfo`) and also decide where autopkgtest or alike might make sense? :)

**Comment by zeha** [comment]
> @zeha yeah, grml-etc-core should get minimized `debian/rules`, and maybe we can look at all our packages together and see which ones might also be targeted towards main Debian (I recall users + customers who are interested in e.g. `grml-hwinfo`) and also decide where autopkgtest or alike might make sense? :)

Okay, I think I got what you're saying, and distilled that into checkbox items in the issue description. Please check if that seems right :)

**Comment by zeha** [comment]
> decide if more packages could be uploaded to Debian

Within this I want to discuss if we can basically get rid of our own repository. For the packages that we don't want to / cant upload we should consider if they need to be packages at all (this relates to turning grml-live into a debian customization deb package builder...).

**Comment by zeha** [comment]
> > decide if more packages could be uploaded to Debian
> 
> Within this I want to discuss if we can basically get rid of our own repository. For the packages that we don't want to / cant upload we should consider if they need to be packages at all (this relates to turning grml-live into a debian customization deb package builder...).

The upload-to-Debian part became https://github.com/grml/grml/issues/220

**Comment by zeha** [comment]
I've opened issues for adding autopkgtests to grml-paste, grml-debootstrap, grml2usb, grml-rescueboot. I think for the keyring we can skip that.

I'd like to close this issue soon, under the assumption that all tasks from it are either done or have dedicated issues.

**Comment by zeha** [comment]
Maybe grml-etc-core should also get the build + release workflows.
